digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/ExportReportDAO.scala

```diff
@@ -0,0 +1,86 @@
+package fi.liikennevirasto.digiroad2.dao
+
+import fi.liikennevirasto.digiroad2.asset.AssetTypeInfo
+import fi.liikennevirasto.digiroad2.oracle.MassQuery
+import fi.liikennevirasto.digiroad2.{ExportStatusInfo, Status}
+import org.joda.time.DateTime
+import org.slf4j.LoggerFactory
+import slick.driver.JdbcDriver.backend.Database.dynamicSession
+import slick.jdbc.{GetResult, PositionedResult}
+import slick.jdbc.StaticQuery.interpolation
+
+class ExportReportDAO {
+  val logger = LoggerFactory.getLogger(getClass)
+
+  lazy val municipalityDao = new MunicipalityDao
+
+  implicit val getResult = new GetResult[ExportStatusInfo] {
+    def apply(r: PositionedResult) : ExportStatusInfo = {
+      val id = r.nextLong()
+      val fileName = r.nextString()
+      val status = r.nextInt()
+      val createdDate = r.nextTimestampOption().map(timestamp => new DateTime(timestamp))
+      val createdBy = r.nextStringOption()
+      val exportedAssets = r.nextString()
+      val municipalities = r.nextString()
+      val content = r.nextStringOption()
+
+      val assetsName = exportedAssets.split(",")
+                                      .map( asset => AssetTypeInfo(asset.toInt).nameFI )
+                                      .mkString(",")
+
+      val municipalitiesAsSet = municipalities.split(",").map(_.toInt).toSet
+      val municipalitiesName = municipalityDao.getMunicipalitiesNameAndIdByCode( municipalitiesAsSet )
+                                              .map(_.name)
+                                              .mkString(",")
+
+
+      ExportStatusInfo(id, status, Status.apply(status).descriptionFi, fileName, createdBy, createdDate, assetsName, municipalitiesName, content)
```

Comment:
Minor thing but I don't think is necessary to specifically call apply, just ``` Status(status).descriptionFi ``` should work the same

Reply:
Applied